### DISCUSSION TITLE
Equipment creation

### DISCUSSION THREAD

**Discussion by b-a-p** [Brewtarget/brewtarget#644] [Q&A] [answered]
Hello,
I want to add my own brewing equipment which is electrical single vessel Klarstein Mundschenk XXL. I can't get of meaning of fields "Kettle top-up water", "Final top-up water" "Lauter deadspace", "Mash Tun Weight", "Specific Heat". Can somebody explain to me what should I put there if my brewing process looks like this:
1. heat water
2. add grains into the mash basket
3. when mashing is done the mash basket is lifted and water is drained into the kettle
4. sparge, wait until drain, remove basket with grains
5. boil
6. cool in the kettle
7. drain wort from kettle to the fermenter.

P.S.:
I also feel it would be very user-friendly to have pop-up help next to each field, but it is nice to have :).

**Comment by mikfire** [comment, marked as answer]
- Kettle top up water: extra water you add to the boil kettle that isn't part of the sparge;
- Final top up water: extra water you add after the boil is complete. This is somewhat common for kit and extract brewers without a big enough boil kettle;
- Lauter deadspace: the amount of water left in the lauter tun that you simply can't get out. This is matters in the old school three vessel systems;
- Mash tun weight -- the weight of your mash tun, required to determine how much heat is lost in warming up the mash tun. This matters if you are using a three vessel system;
- Specific heat -- the specific heat of your mash tun, also required to determine how much heat is lost in warming up the mash tun. This matters if you are using a three vessel system;

It sounds like you are using a single vessel system, so I would expect you could pretty much put 0 in for each and every field and the maths would mostly work.

We used to have roll over help on all those fields. Not quite sure why they've gone away.

**Reply by b-a-p** [reply]
Many thanks @mikfire, maybe help book should be updated with this points.
